src/ekat/logging/ekat_logger.hpp

```diff
@@ -125,9 +125,9 @@ class Logger : public spdlog::logger {
 
   Log::level::level_enum log_level() const {return this->level();}
 
-  Log::level::level_enum console_level() const {return this->get_console_sink()->level();}
+  Log::level::level_enum console_level() {return this->get_console_sink()->level();}
```

Comment:
I think instead of removing const here, you should add const to `get_console_sink` and `get_file_sink`. Those methods return ptrs, so even if someone changes the actual sinks, they won't change the ptrs stored by this class, so it's ok to make them const.

I think someone _might_ call `console_level` at some point from within a const method, and without the const here, that won't work.

Reply:
Thanks @bartgol . This makes sense, and reminds me that I've been staring at my screen too long (should have thought of it myself).